src/forms-opportunities/handler.js

```diff
@@ -15,8 +15,8 @@ import { getRUMDomainkey } from '../support/utils.js';
 import { AuditBuilder } from '../common/audit-builder.js';
 import { wwwUrlResolver } from '../common/audit.js';
 
-const DAILY_THRESHOLD = 200;
-const INTERVAL = 7; // days
+const DAILY_THRESHOLD = 0;
```

Comment:
any reason for setting the daily threshold to 0 ?

Reply:
@05rahulsingh , i think we should keep this threshold.
its just that we should not apply this threshold on form pages but only on the pages having the form CTA for the form oppty: high page views, low form ctr